src/main/cs/RLBotDotNet/RLBotDotNet/Bot.cs

```diff
@@ -0,0 +1,41 @@
+﻿using rlbot.flat;
+
+namespace RLBotDotNet
+{
+    /// <summary>
+    /// Inherit from this class to make a bot.
+    /// Bot logic that should be executed every frame goes in <see cref="GetOutput(GameTickPacket)"/>
+    /// </summary>
+    public abstract class Bot
```

Comment:
Added a field to the Bot class for FieldInfo instead of adding a method in the Bot class since that would require adding a reference to the BotManager instance , which would be icky and the user would be able to mess with the BotManager. Instead, `fieldInfo` is set once during bot instantiation and the user can access that.

Reply:
Bots often initialize while the map is still loading, so this might not go well. We should probably find a way to make that problem easier for all languages.

Reply:
Added a convenience method instead.